modules/microshift-storage-persistent-storage-pv.adoc

```diff
@@ -0,0 +1,281 @@
+requested. Future attributes may include IOPS, throughput, and so on.
+
+[id="pv-access-modes_{context}"]
+== Access modes
+
+A persistent volume can be mounted on a host in any way supported by the
+resource provider. Providers have different capabilities and each PV's
+access modes are set to the specific modes supported by that particular
+volume. For example, NFS can support multiple read-write clients, but a
+specific NFS PV might be exported on the server as read-only. Each PV gets
+its own set of access modes describing that specific PV's capabilities.
+
+Claims are matched to volumes with similar access modes. The only two
+matching criteria are access modes and size. A claim's access modes
+represent a request. Therefore, you might be granted more, but never less.
+For example, if a claim requests RWO, but the only volume available is an
+NFS PV (RWO+ROX+RWX), the claim would then match NFS because it supports
+RWO.
+
+Direct matches are always attempted first. The volume's modes must match or
+contain more modes than you requested. The size must be greater than or
+equal to what is expected. If two types of volumes, such as NFS and iSCSI,
+have the same set of access modes, either of them can match a claim with
+those modes. There is no ordering between types of volumes and no way to
+choose one type over another.
+
+All volumes with the same modes are grouped, and then sorted by size,
+smallest to largest. The binder gets the group with matching modes and
+iterates over each, in size order, until one size matches.
+
+The following table lists the access modes:
+
+.Access modes
+[cols="1,1,3",options="header"]
+|===
+|Access Mode |CLI abbreviation |Description
+|ReadWriteOnce
+|`RWO`
+|The volume can be mounted as read-write by a single node.
+ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
+|ReadOnlyMany
+|`ROX`
+|The volume can be mounted as read-only by many nodes.
+|ReadWriteMany
+|`RWX`
+|The volume can be mounted as read-write by many nodes.
+endif::[]
+|===
+
+[IMPORTANT]
+====
+Volume access modes are descriptors of volume capabilities. They
+are not enforced constraints. The storage provider is responsible for
+runtime errors resulting from invalid use of the resource.
+
+For example, NFS offers `ReadWriteOnce` access mode. You must
+mark the claims as `read-only` if you want to use the volume's
+ROX capability. Errors in the provider show up at runtime as mount errors.
+
+iSCSI and Fibre Channel volumes do not currently have any fencing
+mechanisms. You must ensure the volumes are only used by one node at a
+time. In certain situations, such as draining a node, the volumes can be
+used simultaneously by two nodes. Before draining the node, first ensure
+the pods that use these volumes are deleted.
+====
+
+.Supported access modes for PVs
+[cols=",^v,^v,^v", width="100%",options="header"]
+|===
+|Volume plug-in  |ReadWriteOnce ^[1]^  |ReadOnlyMany  |ReadWriteMany
+|AWS EBS ^[2]^ | ✅ | - |  -
```

Comment:
This should be excluded as well.  Local Volume would be the closest thing to what MicroShift supports.